addons/suggest_locations/controllers/controllers.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+from odoo import http, _
+from odoo.http import request
+
+
+class PutawayStrategy(http.Controller):
+    def _process_suggested_inputs(self, move_line_ids, limit=30, **kwargs):
+        """Process request into a dict"""
+        move_lines = request.env["stock.move.line"].browse(move_line_ids, limit=limit, **kwargs)
```

Comment:
Can we use `StockMoveLine.browse(XXX)` please?

Reply:
Do we need limit on the browse?

Reply:
Is the type of list of int properly handled? when we have identifiers we have to cast them into integers.